source/guides/core-concepts/dashboard-service.md

```diff
@@ -110,7 +110,13 @@ Once you set up your project to record, we generate a unique `projectId` for you
 }
 ```
 
-This helps us uniquely identify your project. If you manually alter this, **Cypress will no longer be able to identify your project or find the recorded builds for it**. We recommend that you check your `cypress.json` including the `projectId` into source control.
+This helps us uniquely identify your project. If you manually alter this, **Cypress will no longer be able to identify your project or find the recorded builds for it**.
+
+We recommend that you check your `cypress.json` including the `projectId` into source control. If you don't want your `projectId` visible in your source code you can set it as an environment variable from your CI provider using the name `CYPRESS_PROJECT_ID`. The exact mechanism for doing so depends on your CI provider but could be as simple as:
+
+```bash
+$ export CYPRESS_PROJECT_ID={projectId}   # replace {projectId} with your actual projectId
+```
```

Comment:
I believe that this change addresses the specific thrust of Gleb's ticket.

It's in the docs for the dashboard as opposed to the FAQ following a steer from Jennifer about where best to put this. (See the comments in #175.)

<img width="788" alt="mention-env" src="https://user-images.githubusercontent.com/1855109/44626414-ac994a00-a913-11e8-9946-535f0e41e56c.png">